dmoj/executors/PAS.py

```diff
@@ -20,12 +20,13 @@ class Executor(CompiledExecutor):
     writeln(line);
 end.
 """
+    flags: List[str] = []
 
     def get_compile_args(self) -> List[str]:
         command = self.get_command()
         assert command is not None
         assert self._code is not None
-        return [command, '-Fe/dev/stderr', '-O2', self._code]
+        return [command, '-Fe/dev/stderr', '-O2', *self.flags, self._code]
```

Comment:
don't you think we need the same thing for `pas-themis`? It wouldn't be the case since the checker won't use that  lang anyway but just to keep things the same